.github/workflows/build-bootc.yaml

```diff
@@ -95,3 +100,63 @@ jobs:
           cosign sign \
             --yes \
             ${{ env.REGISTRY }}/${{ env.REPOSITORY }}/flightctl-agent-${{ matrix.flavor }}@${{ steps.push.outputs.digest }}
+
+      - name: Install podman
+        run: sudo apt install podman
+
+      - name: Build and push disk images
+        run: |
+          mkdir output
+          
+          URL=${{ env.REGISTRY }}/${{ env.REPOSITORY}}/flightctl-agent-${{ matrix.flavor }}
+          
+          oras login -u "${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}" -p "${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}" "${{ env.REGISTRY }}"
+
+          oras blob push $URL .github/workflow-scripts/empty.json
+          cp .github/workflow-scripts/index.json-head index.json
+
+          export SIZE
+          export SHA
+          export ARCH
+          export FORMAT
+
+          for ARCH in amd64 arm64; do
+          for FORMAT in qcow2 raw; do
+          
+            sudo podman run --rm \
+              -it \
+              --privileged \
+              --pull=newer \
+              --security-opt label=type:unconfined_t \
+              -v $(pwd)/output:/output \
+              -v /var/lib/containers/storage:/var/lib/containers/storage \
+              quay.io/centos-bootc/bootc-image-builder:latest \
+                --type $FORMAT \
+                --target-arch $ARCH \
+                "$URL:bootstrap"
+            
+            if [ -f output/image/disk.raw ]; then
+              ARTIFACT=output/image/disk.raw
+            else
+              ARTIFACT=output/qcow2/disk.qcow2
+            fi
+
+            oras blob push "$URL" $ARTIFACT
+            
+            SHA="sha256:"$(sha256sum $ARTIFACT | awk '{print $1}')
+            SIZE=$(cat $ARTIFACT | wc -c)
+            cat .github/workflow-scripts/manifest.json.tmpl | envsubst > manifest.json
+            oras manifest push $URL manifest.json
```

Comment:
Can't we just do 

```
oras manifest push $URL
oras attach $URL --artifact-type application/vnd.diskimage+qcow2 --platform linux/arm64 output/qcow2/disk.qcow2  
```

without need to make/maintain the manifest ourselves?

Reply:
Frank asked for the disk images to be findable by a single multiarch manifest, so I wanted to try to stay true to that. The index vs attach method could just be aesthetic, but I worry that because the Oras docs talk about using attachments as dependencies, it would force users to download all of the disk images for a given flavor in order to access one.

I'm also not sure if we can just attach to an empty manifest list? I tried running the commands you gave above and it gave me 
```
vraiti@vraiti-thinkpadt14sgen2i:~/oras/oras1$ oras manifest push quay.io/vraiti/oras:attach index.json
Pushed [registry] quay.io/vraiti/oras:attach
Digest: sha256:806749360f1c4a2f8d7c94e4674fe5aeb5a494353d71fbc19fc90cae03251b08
vraiti@vraiti-thinkpadt14sgen2i:~/oras/oras1$ oras attach quay.io/vraiti/oras:attach --artifact-type application/vnd.diskimage+qcow2 --platform linux/arm64 amd.txt
Error response from registry: failed to resolve attach: sha256:806749360f1c4a2f8d7c94e4674fe5aeb5a494353d71fbc19fc90cae03251b08: not found: no matching manifest was found in the manifest list
```